src/server.ts

```diff
@@ -8,8 +8,11 @@ import * as fs from 'fs';
 import {
 	IPCMessageReader, IPCMessageWriter, createConnection, IConnection,
 	TextDocuments, InitializeResult, CodeLens, CodeAction, CodeActionKind} from 'vscode-languageserver';
-import { IDependencyCollector, PackageJsonCollector, ReqDependencyCollector, GomodDependencyCollector } from './collector';
+import { GomodDependencyCollector } from './gomod.collector';

```

Comment:
I think we rename all collector and class with some uniformity like file name will be based on ecosystem and class will be specific to manifest file as below. 
1. File ==> maven.collector.ts, npm.collector.ts, pypi.collector.ts and golang.collector.ts
2. Classes can be ==> pomXmlDependencyCollector, packageJsonDependencyCollector, requirementsTxtDependencyCollector and goModDependencyCollector.

Similarly we can rename the test files as well.

Reply:
How about this?

```
requirments.txt/collector.ts
pipfile/requirments.txt
package.json/collector.ts
yarn.lock/collector.ts
pom.xml/collector.ts
go.mod/collector.ts
```